### PR TITLE
Support delaying response

### PR DESCRIPTION
**Pull request by schonert**
This PR includes #15 fix.

### CODE REVIEW

File: src/components/RequestItem/index.js [hunk 1]
```diff
@@ -76,9 +105,11 @@ RequestItem.propTypes = {
     url: PropTypes.string,
     skip: PropTypes.bool,
     method: PropTypes.string,
+    delay: PropTypes.number,
     status: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     response: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
     onToggle: PropTypes.func,
     onStatusChange: PropTypes.func,
     onResponseChange: PropTypes.func,
```
Comment (nutboltu) [inline src/components/RequestItem/index.js:113]: These `onStatusChange` and `onResponseChange` proptypes can be removed.

File: README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ There are few packages those help the developers to mock the backend requests wh
 | `method`   | Supports `GET`, `POST` and `PUT` methods                                                    |          | `GET`   |
 | `status`   | All possible HTTP status codes                                                              |          | `200`   |
 | `response` | JSON format                                                                                 | Y        |         |
-
-> You can change the **status** and **response** from the storybook panel on the fly! :rocket:
+| `delay` | Emulate delayed response time                                                                                  |         | `0`        |
```
Comment (nutboltu) [inline README.md:26]: Let's make it explicit: `Emulate delayed response time in milliseconds`

**Comment by schonert** [comment]
@nutboltu updated the PR resolving comments above

**Review by nutboltu — APPROVED** [review]
Looks good to me! thanks @schonert for the awesome work 👏

**Pull request merged**